experiments/cell/style/index.css

```diff
@@ -0,0 +1,20 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+body {
+  background: white;
+  margin: 0;
+  padding: 0;
+}
+
+.jp-CodeCell {
+  min-height: 50px;
+/*  content-visibility: auto; */
```

Comment:
Are we finally using this or moving on without it?

Reply:
This is a rest from previous attemps. As this brings side effects on the scrollbar, we do not retain that. Moving this out.